utils/zerovec/benches/zerovec_serde.rs

```diff
@@ -101,12 +101,12 @@ fn char_benches(c: &mut Criterion) {
     });
 
     c.bench_function("zerovec_serde/serialize/char/zerovec", |b| {
-        b.iter(|| bincode::serialize(&ZeroVec::from_slice_or_alloc(black_box(ORIGINAL_CHARS))));
+        b.iter(|| bincode::serialize(&ZeroVec::alloc_from_slice(black_box(ORIGINAL_CHARS))));
```

Comment:
Nit: Inside this bench, we don't want to call `alloc_from_slice`. Please move the `alloc_from_slice` outside of the bench function (save the ZeroVec up in the setup section) and black_box the zerovec down here.